products/sle/main.pm

```diff
@@ -762,8 +762,15 @@ elsif (get_var("VIRT_AUTOTEST")) {
         loadtest "virt_autotest/update_package";
         loadtest "virt_autotest/reboot_and_wait_up_normal";
     }
+    elsif (get_var('START_DIRECTLY_AFTER_TEST')) {
```

Comment:
Would you please add a comment here describing why test with START_DIRECTLY_AFTER_TEST shall skip host installation, or describe the whole solution with  START_DIRECTLY_AFTER_TEST, so that others without knowing its details can know the story?

Reply:
comments added: "#Skip host installation for tests run after another test which already installs host on same SUT"